guide/getting-started/9-personal-finance.md

```diff
@@ -0,0 +1,198 @@
+
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/monthly-budget-wallet-setup.png"
+   alt-text = "Alternative text"
+   width = 1600
+   height = 800
+%}
+
+A solution for bitcoin could be a hardware wallet with an accompanying desktop application. While keys are stored and transactions get signed on secure hardware, users can still enjoy the convenience of a desktop application. This avoids or reduces some of the risks involved with the mobile wallet configuration described in the first category, while adding a small amount of friction that is acceptable for transactions that occur less frequently.
+
+## Emergency funds
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/bandaid.png"
+   alt-text = "Alternative text"
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+There are times when we are suddenly faced with a more expensive problem like a medical bill or home repairs, or lose our jobs and need to cover a few months of expenses. For these situations it is recommended to keep around 3-6 months of average monthly expenses. You may or may not end up needing this money, but it’s great to have when you do.
+
+</div>
+
+In traditional finance, this money might get parked in a high-interest savings account or something equivalent that allows for quick low-risk withdrawal but still provides some type of return. Once the fund is set up, the money may not get touched for a long time.
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/savings-wallet-setup.png"
+   alt-text = "Alternative text"
+   width = 1600
+   height = 800
+%}
+
+A bitcoiner may want to set up a cold storage multi signature wallet. In this type of setup, there are multiple applications or hardware devices required to make transactions. So even if one device gets stolen or is locked, the rightful owner still has access to all their funds. Since transactions are infrequent, the balance is tipped towards security over convenience.
+
+## Sinking funds
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/car.png"
+   alt-text = "Alternative text"
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+A sinking fund is a way to systematically set money aside for a planned large expense, like a new car or a down payment for a new home. The amount may be similar to emergency budgets, but the sinking fund is created for a shorter timeline. There are likely to be multiple deposits and potentially only a single withdrawal or transfer after which the fund is no longer needed. This use case does not necessarily required a dedicated account. Based on the amount, it could be covered with a dedicated account in either the wallet for monthly budgeting or savings.
```

Comment:
```suggestion
A sinking fund is a way to systematically set money aside for a planned large expense, like a new car or a down payment for a new home. The amount may be similar to emergency budgets, but the sinking fund is created for a shorter timeline. There are likely to be multiple deposits and potentially only a single withdrawal or transfer after which the fund is no longer needed. This use case does not necessarily require a dedicated account. Based on the amount, it could be covered with a dedicated account in either the wallet for monthly budgeting or savings.
```

Reply:
Hm, I don't see a change... did you remove a space or something along those lines?

Reply:
Oh weird, yeh I think it was just removing some extra space